vignettes_src/workers.Rmd

```diff
@@ -117,7 +117,7 @@ t$wait(10)
 We're going to interact with the rrq object a bit
 
 ```{r}
-rrq <- obj$rrq_controller()
+rrq <- obj$rrq_controller$new(()
```

Comment:
This is unchanged? Because rrq_controller still a function of `queue_didehpc_` object

Reply:
Thanks, great spot. I'll rerun the vignettes too to make sure